openfisca_core/tracers.py

```diff
@@ -130,53 +162,125 @@ def _get_time_in_sec(self) -> float:
     def print_computation_log(self, aggregate = False):
         self.computation_log.print_log(aggregate)
 
-    def generate_performance_graph(self, dir_path):
+    def generate_performance_graph(self, dir_path: str) -> None:
         self.performance_log.generate_graph(dir_path)
 
+    def generate_performance_tables(self, dir_path: str) -> None:
```

Comment:
This and the supporting methods (aggregate_calculation_times, _write_csv) should not be in FullTracer, but in the PerformanceLog class. Keep the responsibilities separate.

Reply:
Done